solara/website/pages/api/task.py

```diff
@@ -0,0 +1,26 @@
+"""# task
+
+Decorator to turn a function or coroutine function into a task.
+A task is a callable that will run the function in a separate thread for normal functions
+or a asyncio task for a coroutine function.
+
+The task callable does will only execute the function once when called multiple times,
+and will cancel previous executions if the function is called again before the previous finished.
+
+
+The wrapped function return value is available as the `.value` attribute of the task object.
+
+
+
+
+"""
```

Comment:
```suggestion
"""# Task

A global way to run code in the background, with the UI available to the user. This is useful for long running tasks, like downloading data.

"""
```

Currently this duplicates the docstring